web/src/main/java/it/pagopa/selfcare/mscore/web/controller/OnboardingController.java

```diff
@@ -341,4 +341,29 @@ public ResponseEntity<byte[]> getOnboardingDocument(@ApiParam("${swagger.mscore.
         }
         return ResponseEntity.ok().headers(headers).body(file.getData());
     }
+
+    /**
+     * The function consume token onboarding request without verify method
```

Comment:
```suggestion
     * Consume onboarding tokens requested without verify method
```

Reply:
this comment is not very clear since the method calls `tokenService.verifyToken` (what do you mean with "without verify method" in this context?)